package.json

```diff
@@ -69,28 +69,28 @@
     "yaml": "^2.3.4"
   },
   "optionalDependencies": {
-    "@codemirror/autocomplete": "^6.16.1",
+    "@codemirror/autocomplete": "^6.16.2",
     "@codemirror/lang-json": "^6.0.1",
-    "@codemirror/lang-yaml": "^6.0.0",
+    "@codemirror/lang-yaml": "^6.1.1",
     "codemirror-json5": "^1.0.3",
     "json5": "^2.2.3"
   },
   "peerDependencies": {
-    "@codemirror/language": "^6.8.0",
-    "@codemirror/lint": "^6.4.0",
-    "@codemirror/state": "^6.2.1",
-    "@codemirror/view": "^6.14.1",
-    "@lezer/common": "^1.0.3"
+    "@codemirror/language": "^6.10.2",
```

Comment:
are we certain users need this version? our example repo seems fine with an earlier version, same for other peers